packages/dialtone-css/lib/build/less/components/notice.less

```
@@ -41,6 +41,51 @@
     box-shadow: var(--notice-box-shadow);
 }
 
+.d-toast-alternate {
+    //  Component CSS Vars
+    //  ------------------------------------------------------------------------
+    --notice-alternate-color-background: var(--dt-color-neutral-white);
+    --notice-alternate-color-text: var(--dt-color-neutral-black);
+    --notice-alternate-color-icon: var(--notice-alternate-color-text);
+    --notice-alternate-padding: var(--dt-space-450);
+    --notice-alternate-font-size: var(--dt-font-size-200);
+    --notice-alternate-line-height: var(--dt-font-line-height-300);
+    --notice-alternate-border-radius: var(--dt-size-500);
+    --notice-alternate-border-color: var(--dt-color-border-subtle);
+    --notice-alternate-border-style: solid;
+    --notice-alternate-border-width: var(--dt-size-100);
+
+    //  Base Styles
+    //  ------------------------------------------------------------------------
+    display: flex;
+    align-items: center;
+    box-sizing: border-box;
+    width: 100%;
```

Use tokens as much as possible.

```suggestion
    width: var(--dt-size-100-percent);
```